docs/data-sources/log_shipping_token.md

```diff
@@ -0,0 +1,36 @@
+# Log Shipping Token Provider
+
+Provides a Logz.io log shipping token resource. This can be used to create and manage Logz.io log shipping tokens.
+
+* Learn more about log alerts in the [Logz.io Docs](https://docs.logz.io/api/#tag/Manage-log-shipping-tokens)
```

Comment:
please replace the "log alerts" with log token or something relevant

Reply:
My bad. fixed.